Lob-API-public.yml

```
@@ -1550,7 +2293,7 @@ tags:
       Lob Dashboard, as well as view your [events](https://dashboard.lob.com/#/events).
       See our [Webhooks Integration Guide](https://lob.com/guides#webhooks_block) for more
       details on how to integrate. Please see the full list of event types available for
-      subscription here.
+      subscription [here](#operation/event).
```

```suggestion
      subscription [here](#tag/Events).
```
Since one level up is where the table it. Sorry -- this might be a remnant change I made...